src/Database/index.js

```diff
@@ -70,9 +77,23 @@ export default class Database {
 
     const sortedOperations: { collection: Collection<*>, operations: CollectionChangeSet<*> }[] = []
     operations.forEach(([type, record]) => {
+      const operationTypeToCollectionChangeType = input => {
```

Comment:
could you move this function to the file top-level (outside the Database class)? or maybe Database/helpers.js if this exists? as far as i can see the only argument is type, so it's better for clarity and performance not to create this function every time .batch() is called.